### PR TITLE
docs(guides): fixes to the Production guide

### PR DESCRIPTION
**Pull request by kyarik**
In the [*Production*](https://webpack.js.org/guides/production/) guide,

- ~~Replaced the confusing clause "We'll use the development one for our `webpack-dev-server`, `npm start`, script" with "We'll use the development one for our `npm start` script", which is consistent with the next clause "and the production one for our `npm run build` script" (notice, it does not say "and the production one for our `webpack`, `npm run build`, script").~~ The sentence about using the new configuration files was rewritten. See https://github.com/webpack/webpack.js.org/pull/3256#discussion_r314993891.
- Improved a sentence about minimizing CSS.

### CODE REVIEW

File: src/content/guides/production.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ Note the use of `merge()` in the environment-specific configurations to easily i
 
 ## NPM Scripts
 
-Now let's repoint our `scripts` to the new configurations. We'll use the _development_ one for our `webpack-dev-server`, `npm start`, script and the _production_ one for our `npm run build` script:
+Now let's repoint our `scripts` to the new configurations. We'll use the _development_ one for our `npm start` script and the _production_ one for our `npm run build` script:
```
Comment (EugeneHlushko) [inline src/content/guides/production.md]:
```suggestion
Now let's point our `scripts` to the new configuration files. `npm start` script, which uses `webpack-dev-server`, at _webpack.dev.js_ and `npm run build` script at _webpack.prod.js_:
```

Comment (kyarik) [reply src/content/guides/production.md]: I think this sentence can be further improved.

What about this:

Now, let's modify our npm scripts to use the new configuration files. For the `start` script, which runs `webpack-dev-server`, we will use `webpack.dev.js`, and for the `build` script, which runs `webpack` to create a production build, we will use `webpack.prod.js`.

?

Comment (EugeneHlushko) [reply src/content/guides/production.md]: amazing, i like it! go ahead

Comment (kyarik) [reply src/content/guides/production.md]: Done!

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**